README.md

```diff
@@ -525,6 +525,33 @@ or with a light image:
 docker run -v /path/to/models:/models ghcr.io/ggerganov/llama.cpp:light -m /models/7B/ggml-model-q4_0.bin -p "Building a website can be done in 10 simple steps:" -n 512
 ```
 
+### Docker With CUDA
+
+Assuming one has the [nvidia-container-toolkit](https://github.com/NVIDIA/nvidia-container-toolkit) properly installed on Linux, or is using a GPU enabled cloud, `cuBLAS` should be accessible inside the container.
+
+#### Building Locally
+
+```bash
+docker build -t local/llama.cpp:full -f .devops/full-cuda.Dockerfile .
+docker build -t local/llama.cpp:light -f .devops/main-cuda.Dockerfile .
```

Comment:
A comment that explains the difference between these would be useful.

Reply:
Added a note describing them, as well renamed their labels to be `cuda-*`.